src/network/networkprotocol.h

```diff
@@ -361,7 +362,14 @@ enum ToClientCommand
 		wstring reason
 	*/
 
-	TOCLIENT_PLAYERITEM = 0x36, // Obsolete
+	// TOCLIENT_PLAYERITEM = 0x36, Obsolete
```

Comment:
just remove the comment here, it's been gone for a long time

Reply:
Done